make/common/NativeCompilation.gmk

```diff
@@ -1063,9 +1063,9 @@ define SetupNativeCompilationBody
           # Setup the command line creating debuginfo files, to be run after linking.
           # It cannot be run separately since it updates the original target file
           $1_CREATE_DEBUGINFO_CMDS := \
-              $$($1_OBJCOPY) --only-keep-debug $$($1_TARGET) $$($1_DEBUGINFO_FILES) $$(NEWLINE) \
-              $(CD) $$($1_SYMBOLS_DIR) && \
-                  $$($1_OBJCOPY) --add-gnu-debuglink=$$($1_DEBUGINFO_FILES) $$($1_TARGET)
+              $$($1_OBJCOPY) --only-keep-debug $$($1_TARGET) $$($1_DEBUGINFO_FILES) $$(NEWLINE)
+          $1_CREATE_DEBUGINFO_LINK_CMDS := $(CD) $$($1_SYMBOLS_DIR) && \
```

Comment:
Maybe add a comment along the lines of "debuglink needs to be done after stripping"